sql/get_trajectories_shark.py

```diff
@@ -0,0 +1,39 @@
+from consts import DB_TABLE
+
+
+def get_trajs_in_frame(fgid, fid, top_k=None):
+    top_string = ''
```

Comment:
Aren't both queries ignoring the I Frame as first frame in the result set?, we are only returning `Ix + Pn` as frames, but not I itself.